cves/CVE-2014-0473.yml

```diff
@@ -60,68 +74,28 @@ bounty:
   url:
 reviews: []
 bugs: []
-repo:
+repo: https://github.com/django/django
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - commit: 1170f285ddd6a94a65f911a27788ba49ca08c0b0
-  note:
+  note: fix for Django version 1.4 added a check in cache middleware to prevent caching responses that set a user-specific cookie in response to a cookie-less request
 - commit: 6872f42757d7ef6a97e0b6ec5db4d2615d8a2bd8
-  note:
+  note: fix for Django version 1.5
 - commit: d63e20942f3024f24cb8cd85a49461ba8a9b6736
-  note:
+  note: fix for Django version 1.6
 - commit: 380545bf85cbf17fc698d136815b7691f8d023ca
-  note:
+  note: fix for Django version 1.7
 vccs:
-- commit: d65526d6886067a8ef368e5b02fce80e1e4c4903
-  note: This VCC was discovered automatically via archeogit.
 - commit: 752659037686026e85c2da52c756d4bfec8c4d13
   note: This VCC was discovered automatically via archeogit.
 - commit: 00fda7f45db2425c1dcb5d927093ede45734d841
   note: This VCC was discovered automatically via archeogit.
-- commit: 7cb140e6d86e9c592983ddb50bfd82480ecec7eb
-  note: This VCC was discovered automatically via archeogit.
-- commit: fe581013b03b3dd38f4893bb5e9dc9695af25d71
-  note: This VCC was discovered automatically via archeogit.
 - commit: ed7a30782b7bf0e30fd47fc407e3b9c22bd80d2f
   note: This VCC was discovered automatically via archeogit.
-- commit: 673e6fc7fb243ed44841b9969d26a161c25733b3
-  note: This VCC was discovered automatically via archeogit.
-- commit: d0eb4693ab479ef0667cd5286ecc89c347b6b2de
-  note: This VCC was discovered automatically via archeogit.
-- commit: f6c991667f6f1ab8d2fd2afc1ee2b5801e78b7e3
```

Comment:
I kept these and either didn't write anything or wrote a comment about why these weren't relevant, but maybe we were supposed to remove them?